modules/ROOT/pages/spell-checking.adoc

```diff
@@ -1,7 +1,7 @@
 = Check spelling in TinyMCE
 
 :title_nav: Spell checking
-:keywords: spell checker spelling browser_spellcheck gecko_spellcheck {productname} provides several options to bring spell checking capabilities to your users.
+:keywords: spell checker spelling browser_spellcheck {productname} provides several options to bring spell checking capabilities to your users.
```

Comment:
Not related to this change but there's actually a conversion issue here by the looks, as most of this isn't keywords.

```suggestion
:keywords: spell checker spelling browser_spellcheck

{productname} provides several options to bring spell checking capabilities to your users.
```

Reply:
looks like the old website omitted the meta description, could we also add one now so we have both keyword and description meta data?

:description: {productname} provides several options to bring spell checking capabilities to your users.